example_configs/tomcat.json

```diff
@@ -0,0 +1,34 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>requestCount:",
```

Comment:
This will only produce 3 metrics, I presume tomcat has far more of interest?


You can do something like:
```
 {
      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):",
      "name": "tomcat_$3_total",
      "labels": {
        "impl":"$1",
        "port":"$2" },
      "help": "Tomcat global $3",
      "type": "COUNTER",
      "attrNameSnakeCase": true
    },
```
to pull out all the metrics